dev/WebView2/WebView2.cpp

```diff
@@ -1505,6 +1505,44 @@ void WebView2::TryCompleteInitialization()
             [this](auto&&, auto&&) { UpdateDefaultBackgroundColor(); });
     }
 
+    // WebView2 in WinUI 2 is a ContentControl that either renders its web content to a SpriteVisual, or in the case that
+    // the WebView2 Runtime is not installed, renders a message to that effect as its Content. In the case where the
+    // WebView2 starts with Visibility.Collapsed, hit testing code has trouble seeing the WebView2 if it does not have
+    // Content. To work around this, give the WebView2 a transparent Grid as Content that hit testing can find. The size
+    // of this Grid must be kept in sync with the size of the WebView2 (see ResizeChildPanel()).
+    AddChildPanel();
+
+    CreateAndSetVisual();
+
+    // If we were recreating the webview after a core process failure, indicate that we have now recovered
+    m_isCoreFailure_BrowserExited_State = false;
+}
+
+void WebView2::AddChildPanel()
+{
+    auto panelContent = winrt::Grid();
+    panelContent.Background(winrt::SolidColorBrush(winrt::Colors::Transparent()));
+    panelContent.Height(this->ActualHeight());
+    panelContent.Width(this->ActualWidth());
+    Content(panelContent);
+}
+
+void WebView2::ResizeChildPanel()
+{
+    auto content = this->Content();
+    if (content)
+    {
+        auto panelContent = content.as<winrt::Grid>();
+        if (panelContent)
+        {
+            panelContent.Height(this->ActualHeight());
+            panelContent.Width(this->ActualWidth());
+        }
+    }
+}
```

Comment:
Looking at this again, can resizing of the Content be handled by MeasureOverride/ArrangeOverride? (Then the Grid will not need to Width/Height specified)

It looks like there was already some code for this in MeasureOverride/ArrangeOverride, but got commented out...